sections/04_study.md

```diff
@@ -586,23 +576,23 @@ for dealing with batch effects [@tag:Shaham2016_batch_effects].
 
 Examining populations of single cells can reveal biologically meaningful subsets
 of cells as well as their underlying gene regulatory networks
-[@tag:Gaublomme2015_th17]. Unfortunately, machine learning generally struggles
+[@tag:Gaublomme2015_th17]. Unfortunately, machine learning methods generally struggle
 with imbalanced data — when there are many more examples of class 1 than class 2 —
 because prediction accuracy is usually evaluated over the entire dataset. To
 tackle this challenge, Arvaniti et al. [@tag:Arvaniti2016_rare_subsets]
 classified healthy and cancer cells expressing 25 markers by using the most
 discriminative filters from a CNN trained on the data as a linear classifier.
-They achieved an impressive precision of 50% to 90% with 80% recall on cells
-where the subset percentage ranged from 0.1 to 1%, which significantly
-outperformed logistic regression and distance-based outlier detection methods.
-However, they did not benchmark against random forests, which tend to be better
-with imbalanced data, and their data was
-fairly low dimensional. Future work will be needed to establish the utility of
+They achieved impressive performance, even for cell types
+where the subset percentage ranged from 0.1 to 1%, significantly
+outperforming logistic regression and distance-based outlier detection methods.
+However, they did not benchmark against random forests, which tend to work better
+for imbalanced data, and their data was
+relatively low dimensional. Future work is needed to establish the utility of
```

Comment:
In light of #495, I don't see how improvements in image classification tell us anything about cell subset identification.  Can we stop the sentence after "cell subset identification."?